framework/src/org/checkerframework/common/subtyping/SubtypingChecker.java

```diff
@@ -28,7 +28,7 @@
  *
  * @checker_framework.manual #subtyping-checker Subtying Checker
  */
-@SupportedOptions( { "quals" })
+@SupportedOptions( { "quals", "qualFolders" })
```

Comment:
This one uses "Folders" another one uses "Dirs". Do we have a precedent for either one? Consistency would be nice.